app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
@denschub Depends on what you mean: I probably need some help with license issues. So essentially, I used the versionist gem: https://github.com/bploetz/versionist . All the files under `app/controllers/api/v0` use it, and in the future `app/controllers/api/v1`, `app/controllers/api/v2`, etc.